File: Telemachus.Api/Enums/Status.cs
using System;

namespace Enums
{
    public class Status
    {
        public static readonly Guid InProgress = new Guid("7DCBAC44-63DC-4425-A41B-D41746E47AA0");
        public static readonly Guid Rejected = new Guid("F740E819-48CE-4FA6-93F3-BCE651865715");
        public static readonly Guid Approved = new Guid("40B413C4-C537-4DDE-A9BB-418815F76A8F");
        public static readonly Guid Completed = new Guid("114F49A8-AAC7-4ABA-8229-374C5AC142CB");
    }
}

File: Telemachus.Web/src/business/status.js
class Status {
  static InProgress = '7dcbac44-63dc-4425-a41b-d41746e47aa0'

  static Completed = '114f49a8-aac7-4aba-8229-374c5ac142cb'

  static Rejected = 'f740e819-48ce-4fa6-93f3-bce651865715'

  static Approved = '40b413c4-c537-4dde-a9bb-418815f76a8f'
}

export const statuses = Object.entries(Status)
  .filter(([_label, value]) => !Array.isArray(value))
  .map(([label, value]) => ({
    label,
    value
  }))

export const getStatusLabel = status => statuses.find(e => e.value === status)?.label ?? null

export default Status
